addons/radar/XEH_preInit.sqf

```diff
@@ -80,6 +80,23 @@ if (_acre || _tfar) then {
     GVAR(showSpeaking_replaceIcon) = false;
 };
 
+if (isClass (configfile >> "CfgPatches" >> "ace_medical_engine")) then {
+  [
+    QGVAR(ace_medic),
+    "CHECKBOX",
+    [localize LSTRING(ace_medic), localize LSTRING(ace_medic_desc)],
+    [CBA_SETTINGS_CAT, _curCat],
+    true,
+    true,
+    {
+      [QGVAR(refreshUI),[]] call CBA_fnc_localEvent;
```

Comment:
Not needed. The cache loop will query icon changes, even with changed settings midgame.